devices.js

```diff
@@ -12532,6 +12532,23 @@ const devices = [
             await readMeteringPowerConverterAttributes(endpoint);
             await configureReporting.instantaneousDemand(endpoint);
         },
+        onEvent: async (type, data, device) => {
```

Comment:
Question: why not do this in `configure()`?

Reply:
> We don’t want to restore the bindings if someone runs configure on it via the api.
> Well, add them to the _binds property again, as we’re just recording the devices defaults... which are only there after joining until the user unbind them.
> 
> I did not find a way to detect if the device was in interviewing mode from configure.
> 
> If we drop the configureKey and can ensure configure is only run once unless your remove/repair a device this can be moved. But we’re not there yet.
> 

From the e-mail, so we can resolve this cleanly for github